src/platform/site-wide/va-footer/components/Footer.jsx

```diff
@@ -5,6 +5,7 @@ import DesktopLinks from './DesktopLinks';
 import MobileLinks from './MobileLinks';
 import { createLinkGroups } from '../helpers';
 import { replaceWithStagingDomain } from '../../../utilities/environment/stagingDomains';
+import './../../../monitoring/analytics';
```

Comment:
The `monitoring` dir seems like a good folder for this, but this means the analytics will execute on every page across the website, including React apps, so there may be two sets of analytics executing (these, and the app's specific analytics).

I had the analytics added into `app/static-pages` because `static-pages` runs on content pages, but not React apps.

Reply:
If we aren't worried about that though, then I would just do this import at `src/platform/site-wide/index.js` instead of the footer.